spec/api-browser-window-spec.ts

```diff
@@ -6490,8 +6490,7 @@ describe('BrowserWindow module', () => {
       expect(w.getBounds()).to.deep.equal(newBounds);
     });
 
-    // FIXME(codebytere): figure out why these are failing on macOS arm64.
-    ifit(process.platform === 'darwin' && process.arch !== 'arm64')('should not display a visible background', async () => {
+    ifit(hasCapturableScreen())('should not display a visible background', async () => {
```

Comment:
This test is reliably failing on macOS arm64: https://github.com/electron/electron/actions/runs/9943722345/job/27512304857?pr=42320#step:14:686